src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/helpers/PermissionChecker.kt

```diff
@@ -73,7 +73,7 @@ class PermissionChecker(
             val scope = permission.scope as Scope.Specific
 
             println("Checking that same permission with different scope will not satisfy the requirement")
-            val badPermission = ReifiedPermission(permission.name, Scope.Specific(scope.scopePrefix, "bad-id"))
+            val badPermission = ReifiedPermission(permission.name, Scope.Specific(scope.databaseScopePrefix, "bad-id"))
```

Comment:
Not sure why/how this change made it in, but there is no longer a `val` called `scopePrefix` in `Scope.Specific`

Reply:
Weird